pkg/local/sync.go

```diff
@@ -277,7 +277,8 @@ func (s *SyncManager) upload(ctx context.Context, rootPath string, remote *uri.U
 	if err := fileutil.VerifySafeFilename(source); err != nil {
 		return err
 	}
-	dest := strings.TrimPrefix(filepath.ToSlash(fmt.Sprintf("%s%s%s", remote.GetPath(), uri.PathSeparator, path)), uri.PathSeparator)
+	remotePath := strings.TrimRight(remote.GetPath(), uri.PathSeparator)
```

Comment:
Regarding this change, see [this PR](https://github.com/treeverse/lakeFS/pull/7965).